packages/main/src/types/TabContainerBackgroundDesign.js

```diff
@@ -0,0 +1,41 @@
+import DataType from "@ui5/webcomponents-base/dist/types/DataType.js";
+
+/**
+ * Background design for the header and content of Tab Container
+ * @lends sap.ui.webcomponents.main.types.TabContainerBackgroundDesign.prototype
+ * @public
+ */
+const TabContainerBackgroundDesigns = {
+	/**
+	 * A Solid background color.
+	 * @public
+	 * @type {Solid}
+	 */
+	Solid: "Solid",
+
+	/**
+	 * A Transparent background color.
+	 * @public
+	 * @type {Transparent}
+	 */
+	Transparent: "Transparent",
+};
+
+/**
+ * @class
+ * Defines the Tab Container's header and content background color
```

Comment:
Because the background color of the header and the content are controlled separately I suggest:
* Defines the Tab Container's header and content background colors